async.md

```diff
@@ -0,0 +1,230 @@
+The ClearBlade Async library works with the event loop to provide asynchronous ClearBlade functions. All functions return promises and execute asynchronously. The `init` function is not required. 
+
+# Usage
+
+1. __[Query](#query)__
+2. __[Collection](#collection)__
+
+# Query 
+
+Class: ClearBladeAsync.Query(options)
+
+This class allows for the execution of async ClearBlade query functions and returns promises. The functions and examples are similar to the [ClearBlade Query Class](https://github.com/ClearBlade/native-libraries/blob/master/clearblade.md#query)
+
+To instantiate the async query class just call:
+
+~~~javascript
+	var query = ClearBladeAsync.Query("Activity")
+~~~
+
+# Collection
+
+Class: ClearBladeAsync.Collection(options)
+
+This class allows for the execution of async ClearBlade collection functions and returns promises. The `init` function is not required.
+
+To instantiate the async class just call:
+
+~~~javascript
+	var collection = ClearBladeAsync.Collection(nameorID)
+~~~
+## ClearBladeAsync.newCollection(name)
+
+Creates a new ClearBlade collection.
+
+* @param {string} name
+* @returns {Promise}
+
+~~~javascript
+ ClearBladeAsync.newCollection("<COLLECTION_NAME>")
+~~~
+
+## ClearBladeAsync.Collection(nameOrID)
```

Comment:
Is this documenting the Collection object instantiation, or the fetch function? The descriptions describes fetch, the partial jsdoc describes object instantiation, and the example looks like it's fetch again.